### DISCUSSION TITLE
Stuck on Configuring Remote Session / 'Welcome'

### DISCUSSION THREAD

**Discussion by EmuLabs** [kimmknight/remoteapptool#119] [Q&A]
Hi Dev and community.
Both client and 'server' running Win10 Pro 22H2.
I have tried a variety of config on the RDP/remoteapp shortcut.
For example if already logged into host, I run the remoteapp shortcut I can interact with session to log off existing user, but then as soon as I get to 'Welcome' it just hangs.
I try variety of apps but currently testing with notepad. Please let me know what info I can provide to help. Thank you
![image](https://github.com/kimmknight/remoteapptool/assets/105586721/0328fd4e-2a1d-4a28-803f-4a47c4dbc796)

**Comment by MrBrianGale** [comment]
Is someone currently logged into the "server" when you get stuck on the welcome screen?
With desktop OS's (such as Windows 10), you can only have 1 active session at a time. If someone else is logged in, the system is likely waiting for the other user to confirm that it is OK to disconnect the session and start the second session.

**Reply by EmuLabs** [reply]
Hi mate,

I have tried with nothing signed in and signed in. When I use remoteapp with person signed in, I can interact with the session to force them to disconnect. As soon as I do that session freezes. When use remoteapp with no one signed in, session freezes straight away.

**Comment by EmuLabs** [comment]
Fix by enabling the Remote Desktop Services UserMode Port Redirector. Was previously set to disabled.
![image](https://github.com/kimmknight/remoteapptool/assets/105586721/d0fb39b1-cca3-41a5-b124-1979cc5f1dc0)
